test/test_transformer.cc

```diff
@@ -518,3 +518,211 @@ TEST(Transformer, Codegen) {
   ASSERT_TRUE(loop4pos > loop3pos);
   ASSERT_TRUE(stmtpos > loop4pos);
 }
+
+static isl::schedule_node tile(isl::schedule_node node,
+			       std::vector<int> tileSizes,
+  			       int defaultSize = 32) {
```

Comment:
I find this interface quite confusing, I would have made it fail on an assertion (it's a test after all!) if the size of `tileSizes` mismatches the number of band members.  Try writing documentation for this function to understand my complaint :)

Reply:
Ok, I think I got your point. In this case, if I want to use assert in the method I should you cassert or the one provided by googletest? It is ok to mix both?

Reply:
You cannot use those from googletest unless you are inside a function declared as test, so just use the C one.  Or have a manual check and call `ISLUTILS_DIE` from `die.h`.